app/models/app_group.rb

```diff
@@ -48,6 +49,35 @@ class AppGroup < ApplicationRecord
     )
   }
 
+  scope :search_by_labels, ->(query) {
+    return nil if query.blank?
+
+    queries = []
+    values = []
+    query.each_pair do |k,v|
+      if k.blank? || v.blank?
+        next
+      end
+
+      if k.to_s.starts_with?("keys_")
+        queries.append("app_groups.labels->>? LIKE ?")
+      end
+
+      if k.to_s.starts_with?("values_")
+        values.append("%#{v}%")
+      else
+        values.append(v)
+      end
+    end
+
+    puts sprintf("%s: %s", queries.join(' AND '), values)
```

Comment:
do we still need this one or we need to store it to the log?

Reply:
will remove it 🙇

Reply:
removed on https://github.com/BaritoLog/BaritoMarket/pull/270/commits/0e200116945d73ecd77a258686742bfc26022047